Content/Common/common.json

```diff
@@ -27,5 +27,7 @@
 	"common_component": "Component",
 	"common_unauthorized": "Unauthorized",
 	"common_rateLimitReached": "Gateway access blocked due to exceeding rate limit. Please wait a few minutes to retry.",
-	"common_badHttpResponseStatusCode": "Bad HTTP response status code %d"
+	"common_badHttpResponseStatusCode": "Bad HTTP response status code %d",
+	"common_worth": "Worth",
+    "common_unknown": "Unknown"
```

Comment:
Matt generally wants to avoid having common strings, but I guess we can keep them here and specify the key more, it's the stake claim NFT worth, right? And make one for LSUs too.

Reply:
actually we have plenty of `Worth` strings defined already. Will define it in account history section then